readme.md

```diff
@@ -469,6 +480,61 @@ The following file types will not be accepted:
 - `.csv` - [Reason.](https://github.com/sindresorhus/file-type/issues/264#issuecomment-568439196)
 - `.svg` - Detecting it requires a full-blown parser. Check out [`is-svg`](https://github.com/sindresorhus/is-svg) for something that mostly works.
 
+## Custom detectors
```

Comment:
I suggest to move it before [_Supported File Types_](https://github.com/FredrikSchaefer/file-type/tree/extend-file-type-updated#supported-file-types), so that the API documentation is not fragmented.

Reply:
Done.